manifests/oci-flexvolume-driver.yaml

```diff
@@ -31,6 +32,9 @@ spec:
             - mountPath: /tmp
               name: config
               readOnly: true
+            - mountPath: /tmp2
```

Comment:
An aside (to this PR) but do we really want to mount in /tmp vs something else? (above).

Reply:
honestly i don't know, i was just following suit

Reply:
Separate discussion maybe.